_episodes/08-development-setup.md

```diff
@@ -53,7 +52,13 @@ Let’s get started.
 
 The ESMValTool source code is available on a public GitHub repository:
 [https://github.com/ESMValGroup/ESMValTool](https://github.com/ESMValGroup/ESMValTool).
-To obtain the code, the easiest way is to clone the repository:
+To obtain the code, there are two options:
+
+1. download the code from the repository. A ZIP file called
+   ``ESMValTool-master.zip`` is downloaded. To continue the installation, unzip
+   the file, move to the ``ESMValTool-master`` directory and then follow step
```

Comment:
the file, move to the ``ESMValTool-master`` directory and then follow the sequence of steps

Reply:
starting from **2 ESMValTool dependencies**. Note that you have to be in the ``ESMValTool`` directory to complete the installation successfully.

Reply:
> starting from **2 ESMValTool dependencies**. 

The sentence is revised as you suggested.

> Note that you have to be in the `ESMValTool` directory to complete the installation successfully.

When users download the source code, the folder called `ESMValTool-master` and not `ESMValTool` anymore. It is already mentioned that "_move to the ESMValTool-master directory and then follow the sequence of steps..._". Therefore, users can follow the next instructions without being worried about the working directory.